sphinx-doc/module-md-force.rst

```diff
@@ -8,14 +8,24 @@ md.force
 .. autosummary::
     :nosignatures:
 
+<<<<<<< HEAD
```

Comment:
Please resolve these merge conflicts. Accept the changes from `feature/new-object-API`.